dts/bindings/dali/zephyr,dali-pwm.yaml

```
@@ -0,0 +1,63 @@
+description: PWM based DALI interface
+
+compatible: "zephyr,dali-pwm"
+
+include: base.yaml
+
+properties:
+  pwms:
+    type: phandle-array
+    required: true
+    description: |
+      The PWM unit will control the transmission of DALI frames.
+
+  counter:
+    type: phandle
+    required: true
+    description: |
+      The counter will evaluate the received DALI frames.
+
+  chan-id-stop:
+    type: int
+    default: 0
+    description: |
+      Channel for Rx-stopbit timeout.
+
+  chan-id-collision:
+    type: int
+    default: 1
+    description: |
+      Channel for Tx collision detection.
+
+  chan-id-tx:
+    type: int
+    default: 2
+    description: |
+      Channel for Tx re-programming.
+
+  rx-gpios:
+    type: phandle-array
+    required: true
+    description: |
+      GPIO pin that connects to the DALI Rx pin.
+
+  tx-flank-shift-us:
+    type: int
+    description: |
+      Difference in propagation delay in ns between the falling and rising edge.
+      If for example the high to low delay is 8 us and low to high is 22 us, then enter
+      (-14) here. Keep in low µs range.
+
+  rx-flank-shift-us:
+    type: int
+    description: |
+      Difference in propagation delay in ns between the falling and rising edge.
+      If for example the high to low delay is 47 us and low to high is 15 us, then enter
+      (32) here. Keep in low µs range.
```

Above its us, not µs.
```suggestion
      (32) here. Keep in low us range.
```